src/context/CompleteContext.jsx

```diff
@@ -0,0 +1,52 @@
+import { createContext, useContext, useState } from "react";
```

Comment:
Creating contexts for your todo and for your completed was smart! It's easy to understand your code this way. 👯 It makes total sense in your layout too. Cool!